### ISSUE TITLE
iptv for "MediaFusion"

### ISSUE THREAD

**Issue by SukunaVirus** [mhdzumair/MediaFusion#77]
Can you add this iptv portal to the MediaFusion add-on?

Panel Url ➤ http://weaseltv.live/c/
Mac Address ➤ 00:1A:79:8b:63:c1

**Comment by mhdzumair** [comment]
Is this link working? Im not able to access. What about the mac address?

**Comment by SukunaVirus** [comment]
> Is this link working? Im not able to access. What about the mac address? 

Yes it's working. But you can add iptv portals? I tried in stbemu app. It will be easy if it's in the stremio app !

**Comment by SukunaVirus** [comment]
> Is this link working? Im not able to access. What about the mac address? 

And this works fine too: 

http://entertainmentj4u.com/stalker_portal/c

00:1A:79:15:A3:D1

**Comment by mhdzumair** [comment]
> > Is this link working? Im not able to access. What about the mac address?
> 
> And this one works fine too:
> 
> http://entertainmentj4u.com/stalker_portal/c 00:1A:79:15:A3:D1

![image](https://github.com/mhdzumair/MediaFusion/assets/44891939/78d2579a-6602-4bc4-b763-65b1d7f3ae1b)

this is what i see

**Comment by SukunaVirus** [comment]
> > > Is this link working? Im not able to access. What about the mac address?
> > 
> > And this one works fine too:
> > 
> > http://entertainmentj4u.com/stalker_portal/c 00:1A:79:15:A3:D1
> 
> ![image](https://github.com/mhdzumair/MediaFusion/assets/44891939/78d2579a-6602-4bc4-b763-65b1d7f3ae1b)
> 
> this is what i see

Or maybe try with "OTT Navigator" it's working for me !

![Screenshot_2024_0113_143724.jpg](https://github.com/mhdzumair/MediaFusion/assets/144272091/64526c60-470e-43d0-9c3c-5dcd78ea3645)
